recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -60,3 +61,7 @@ do_compile() {
            "
 }
 
+# Let do_build task be executable for this recipe
+python() {
+    d.delVarFlag("do_build","noexec")
```

Comment:
I believe this can be moved to bbclass: for Android it will do what we need and will do no harm to the recipes which already have do_build task executable. This way users don't care to put this code into their recipes

Reply:
And, of course, space after comma

Reply:
Done